app/views/groups/edit.html.erb

```diff
@@ -6,6 +6,15 @@
 
 <%= render "modal-edit", group: @group %>
 
+<div style="
+  overflow:auto;
+  word-break: break-word;
+  white-space: pre-line;
+  max-width: 45%;
+  max-height:500px;">
```

Comment:
Should CSS be done this way?

Reply:
There should be bootstrap classes for them that fit your needs.
Maybe you could also substitute CSS percentages, as they are also not responsive for mobile. In the wireframe it looks like a grid of 4:8